website/docs/d/nsxv_dnat.html.markdown

```diff
@@ -0,0 +1,41 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_nsxv_dnat"
+sidebar_current: "docs-vcd-data-source-nsxv-dnat"
+description: |-
+  Provides a vCloud Director DNAT data source using proxied NSX-V API. This can be used to read
```

Comment:
@lvirbalas Question here what is more understandable for our users -> NSX-V or advanced edge gtw? Note is ok, but major description should aim for best explanation and Note should add additional info.

Reply:
Changed it to mention advanced edge gateways and only (NSX-V)